### PR TITLE
feat: added language-switch to footer

### PR DESCRIPTION
**Pull request by WiebkeFr**
pr contains:
-  language-switch added to footer 
- color change of footer

to test:
[https://satellytescommain-featlanguageswitchinfooter.gtsb.io/](https://satellytescommain-featlanguageswitchinfooter.gtsb.io/)

### CODE REVIEW

File: src/components/icons/social/github.tsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import styled from 'styled-components';
 
+interface GithubProps {
```
Comment (feedm3) [inline src/components/icons/social/github.tsx:4]: Thats only for the `GithubIconWrapper` right? Then the naming should be `GithubIconWrapperProps`. The interface should also be right above the implementation that you can see with one sight what the props are.

File: src/components/icons/social/github.tsx [hunk 2]
```diff
@@ -15,11 +18,12 @@ const GithubIcon: React.FC = (props) => {
...
 
-export const GithubIconWrapper = styled(GithubIcon)`
+export const GithubIconWrapper = styled(GithubIcon)<GithubProps>`
   transition: fill 0.2s ease-in;
-  fill: #202840;
+  fill: #ffffff;
 
   &:hover {
     fill: #ffffff;
+    opacity: 1;
```
Comment (feedm3) [inline src/components/icons/social/github.tsx:27]: props are not used here?

File: src/components/icons/social/linkedin.tsx [hunk 3]
```diff
@@ -22,8 +23,9 @@ export const IconLinkedIn: React.FC<LinkedinProps> = (props) => (
 
 export const LinkedinWrapper = styled(IconLinkedIn)<LinkedinProps>`
   transition: fill 0.2s ease-in;
-  fill: ${(props) => (props.color ? props.color : '#202840')};
+  fill: ${(props) => (props.color ? props.color : '#FFFFFF')};
   &:hover {
     fill: ${(props) => (props.color ? props.color : '#ffffff')};
+    opacity: 1;
```
Comment (feedm3) [inline src/components/icons/social/linkedin.tsx:29]: also `opacity` prop is not used here

**Review by feedm3 — CHANGES_REQUESTED** [review]

File: src/components/icons/social/github.tsx (hunk 1 above)
Comment (WiebkeFr) [reply src/components/icons/social/github.tsx:4]: It is not used in `GithubIconWrapper` but in the component `IconGithub` itself with `{...props}` 
(just like in `IconLinkedIn`)
Shall I write it extra with `opacity={props.opacity}`?

**Review by feedm3 — APPROVED** [review]

**Pull request merged**